src/psmove.c

```diff
@@ -560,25 +560,49 @@ psmove_connect_internal(wchar_t *serial, char *path, int id)
 
 #ifdef _WIN32
     /* Windows Quirk: USB devices have "0" as serial, BT devices their addr */
-    if (serial != NULL && wcslen(serial) > 1) {
-        move->is_bluetooth = 1;
-    }
-    /* Windows Quirk: Use path instead of serial number by ignoring serial */
-    serial = NULL;
-
-    /* XXX Ugly: Convert "col02" path to "col01" path (tested w/ BT and USB) */
-    char *p;
-    psmove_return_val_if_fail((p = strstr(path, "&col02#")) != NULL, NULL);
-    p[5] = '1';
-    psmove_return_val_if_fail((p = strstr(path, "&0001#")) != NULL, NULL);
-    p[4] = '0';
-#endif
+	if (serial != NULL && wcslen(serial) > 1) {
+		move->is_bluetooth = 1; // Is this used anywhere else?
+	}
+	serial = NULL;  // Set the serial to NULL, even if BT device, to use psmove_get_serial below.
+    
+	/*
```

Comment:
Also here in this comment block, please be consistent with indentation and other multi-line comment blocks in this file.